ydb/docs/en/core/changelog-cli.md

```diff
@@ -1,5 +1,19 @@
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.17.0 {#2-17-0}
+
+Released on December 4, 2024. To update to version **2.17.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+### Features
+
+* Improved performance of parallel [importing data from the file system](./reference/ydb-cli/export-import/tools-restore.md) using the `{{ ydb-cli }} tools restore` command.
```

Comment:
We'd better use the same categories for cli and server, and this sounds more like performance improvement.

Reply:
Added performance section as in the server